kong/db/schema/entities/routes.lua

```diff
@@ -159,7 +159,7 @@ local routes = {
 
       { snis = { type = "set",
                  description = "A list of SNIs that match this Route.",
-                 elements = typedefs.sni }, },
+                 elements = typedefs.wildcard_host }, },
```

Comment:
It will apply to 'traditional' flavor, but I think it will ok

Reply:
I exclude the `traditional` flavor now.